stakewise_cli/web3signer.py

```diff
@@ -43,38 +47,39 @@ def keys(self) -> List[DatabaseKeyRecord]:
 
         keys_count = len(self.operator_deposit_data_public_keys)
         index = 0
-        with click.progressbar(
-            length=keys_count,
-            label="Syncing key pairs\t\t",
-            show_percent=False,
-            show_pos=True,
-        ) as bar:
-            while len(deposit_data_key_records) < keys_count:
-                curr_progress = len(deposit_data_key_records)
-
-                signing_key = get_mnemonic_signing_key(self.mnemonic, index, IS_LEGACY)
-                public_key = Web3.toHex(G2ProofOfPossession.SkToPk(signing_key.key))
-                public_key = add_0x_prefix(public_key)
-
-                private_key = str(signing_key.key)
-                encrypted_private_key, nonce = self.encoder.encrypt(private_key)
-
-                key_record = DatabaseKeyRecord(
-                    public_key=public_key,
-                    private_key=bytes_to_str(encrypted_private_key),
-                    nonce=bytes_to_str(nonce),
-                    validator_index=index // self.validator_capacity,
+        click.secho("Syncing key pairs...", bold=True)
+        while True:
+            signing_key = get_mnemonic_signing_key(self.mnemonic, index, IS_LEGACY)
+            public_key = Web3.toHex(G2ProofOfPossession.SkToPk(signing_key.key))
+            public_key = add_0x_prefix(public_key)
+
+            if len(deposit_data_key_records) >= keys_count:
```

Comment:
I think we overcomplicate logic here. We should add a key in case any of the following holds:
1) it's registered and validator status is not exited or withdrawn.
2) the public key is in `operator_deposit_data_public_keys`

Otherwise, we break